pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -428,16 +428,34 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 	targetVMName := mapper.ResolveVMName(instance.Spec.TargetVMName)
 
 	// Define VM spec
+	// Update condition to VM template matching:
+	processingCond := conditions.NewProcessingCondition(string(v2vv1alpha1.VMTemplateMatching), "Matching virtual machine template", corev1.ConditionTrue)
+	if err := r.upsertStatusConditions(instanceNamespacedName, processingCond); err != nil {
+		return "", err
+	}
 	template, err := provider.FindTemplate()
 	var spec *kubevirtv1.VirtualMachine
+	config, cfgErr := r.kvConfigProvider.GetConfig()
+	if cfgErr != nil {
+		log.Error(cfgErr, "Cannot get KubeVirt cluster config.")
+	}
 	if err != nil {
 		reqLogger.Info("No matching template was found for the virtual machine. Using empty VM definition")
+		if !config.ImportWithoutTemplateEnabled() {
```

Comment:
The message above should be split into 2 parts, the first part - should remain there as:
`reqLogger.Info("No matching template was found for the virtual machine.")`

and the second part after this condition ends successfully, before line 449:
`reqLogger.Info("Using empty VM definition")`

The reason is to avoid the message of "Using empty VM definition" if this isn't supported by the setup.

Reply:
Done.